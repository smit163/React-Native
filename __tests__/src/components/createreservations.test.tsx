import React from 'react';
import MakeReservation from '../../../src/components/createreservations';
import Enzyme, { shallow } from 'enzyme';


describe('Testing ReassignLocationMenu component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <MakeReservation />
    );
    expect(wrapper).toMatchSnapshot();
    wrapper.setProps({ count: 1 });
    expect(wrapper).toMatchSnapshot();
  });
})