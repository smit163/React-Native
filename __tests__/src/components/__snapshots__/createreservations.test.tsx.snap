// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ReassignLocationMenu component renders as expected 1`] = `ShallowWrapper {}`;

exports[`Testing ReassignLocationMenu component renders as expected 2`] = `ShallowWrapper {}`;
